### PR TITLE
release: v0.2.5 with ListRelations and improved parallization

### PR DESCRIPTION
**Pull request by rhamzeh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v0.2.5
+
+### [0.2.5](https://github.com/openfga/js-sdk/compare/v0.2.4...v0.2.5) (2023-04-21)
+
+- feat(client): implement `listRelations` to check what relationships a user has with an object
+- feat(client): proper parallel limit for batch fns (BatchCheck, etc..)
```
Comment (adriantam) [inline CHANGELOG.md:8]: is this a fix or feat?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+- chore(ci): publish provenance data
+- chore(deps): update dependencies
+
```
Comment (adriantam) [inline CHANGELOG.md:11]: Do we also want to call out that we expect schema version be returned (i.e., it will not work with old version of OpenFGA)?

**Review by adriantam — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (rhamzeh) [reply CHANGELOG.md:11]: Will add it

File: CHANGELOG.md (hunk 1 above)
Comment (rhamzeh) [reply CHANGELOG.md:8]: Somewhere between both? But I'm on the side of feature, parallelization instead of batching. Will make it a fix.

**Review by adriantam — APPROVED** [review]

**Pull request merged**